### PR TITLE
Cosmetic fixes

### PR DESCRIPTION
**Pull request by IuriiVM**

### CODE REVIEW

File: 020_IoCWithUnity/Bootstrapper.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public static class Bootstrapper
         public static void SetupContainer(IUnityContainer unityContainer)
         {
             // TODO: Register your dependencies here
-            // unityContainer.RegisterType<IInterface, Implementor>(...)
+            // unityContainer.RegisterType<IInterface, Implementor>();
```
Comment (voloda) [inline 020_IoCWithUnity/Bootstrapper.cs]: Please revert this change. The original purpose was to demonstrate overloads of the method

**Comment by voloda** [comment]
Thanks Iurii - I would just ask you for a single small change above. Also please make similar changes into [https://github.com/IoCHowTo/IoCTasksSolutions] to keep sample solutions in sync.

**Comment by IuriiVM** [comment]
Hi Vlado,

I made needed adjustments and added some more wording fixes. Changes in
IoCTasksSolutions were made as well.

Best regards,
Yura

**Comment by voloda** [comment]
:+1:
